DigitalLibraryofTennessee/DLTN_XSLT
# Address DLTN-231.

**GitHub Issue**: [DLTN-231](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/231)

## What does this Pull Request do?

Regenerates all current catalogs and adds one for MTSU.

## How should this be tested?

Run a transform for a set versus a record that should have a iiif-manifest with an updated catalogs directory.

## Interested parties

@mlhale7
